sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py

```diff
@@ -115,48 +128,36 @@ def __aiter__(self):
         raise last_exception
 
     def _create_handler(self):
-        alt_creds = {
-            "username": self._client._auth_config.get("iot_username") if self._redirected else None,  # pylint:disable=protected-access
-            "password": self._client._auth_config.get("iot_password") if self._redirected else None  # pylint:disable=protected-access
-        }
-
         source = Source(self._source)
         if self._offset is not None:
             source.set_filter(self._offset._selector())  # pylint:disable=protected-access
+
+        if StrictVersion(uamqp.__version__) < StrictVersion("1.2.3"):  # backward compatible until uamqp 1.2.3 released
+            desired_capabilities = {}
```

Comment:
I think we can log a warning here saying:
"The capability of tracking last enqueued event properties requires uamqp >= 1.2.3, current uamqp version {} doesn't support the feature.".format(uamqp.__version__).

Reply:
This will only exist for very short time before uamqp1.2.3 is released. Users won't have this when we release eventhubs preview4 because uamqp1.2.3 is required.